pkg/trust/artifacts.go

```diff
@@ -162,6 +164,16 @@ func UpdateShim(inShim, newShim, keysetPath string) error {
 }
 
 func SetupBootkit(keysetName string) error {
+	// TODO - we have to fix this by
+	// a. having bootkit generate arm64
+	// b. changing the bootkit layer naming to reflect arch
+	// c. using the bootkit api here instead of doing it ourselves
+	// for now, we just do nothing on arm64
+	if runtime.GOARCH != "amd64" {
+		log.Warnf("Running on %q, so not building bootkit artifacts (only amd64 supported).", runtime.GOARCH)
+		return nil
```

Comment:
Just curious... why don't we want to error out? Do we have to create arm bootkit manually so just need the keyset?

Reply:
We run our testcases on arm.  If we error out on arm, our testcases will always fail.

This way, we will not have bootkit artifacts on arm64, which just means we can't autocreate our isos on arm64.  which is fine, bc we can't do that anyway.